src/common/phy_stats.cc

```diff
@@ -9,6 +9,9 @@
 
 #include "logger.h"
 
+//set the number of recorded subcarrier data samples
+static constexpr size_t kNumRecSc = 4;
```

Comment:
Why is this 4?  why would you set it to anything else.  does this make sense to be hard coded?